andela/ah-lobos-frontend
# #166789844 View and read notifications

#### What does this PR do?
This PR will create UI component to:
- Users should be able to view notifications

#### Description of Task to be completed?
- Users should be able to 
  - view notifications that are relevant to them.
  - see a _badge_ UI component that shows how many new notifications they have

#### How should this be manually tested?
First `git clone` https://github.com/andela/ah-lobos-frontend.git

Then :
navigate into the project directory
>cd ah-lobos-frontend 

 install all dependencies

>npm install

run test for end-to-end

>npm run cypress

run test for unit test

>npm run test

start application

>npm start

- Your default browser will open, click ```notification icon``` on the right side of the ```navigation bar```, then you will see your notification

#### Any background context you want to provide?
- You need to be logged in the system(have some notifications)
#### What are the relevant pivotal tracker stories?
[#166789844](https://www.pivotaltracker.com/story/show/166789844)
#### Screenshots (if appropriate)
 
<img width="1440" alt="Screen Shot 2019-09-18 at 11 30 32" src="https://user-images.githubusercontent.com/30442301/65136406-ccf8e280-da07-11e9-9aad-6c5e215de228.png">

<img width="1440" alt="Screen Shot 2019-09-18 at 11 28 16" src="https://user-images.githubusercontent.com/30442301/65136457-e437d000-da07-11e9-8cf2-af5f15ddef4e.png">


#### Questions:
- N/A
